assets/src/edit-story/app/canvas/useCanvasCopyPaste.js

```diff
@@ -98,16 +98,25 @@ function useCanvasGlobalKeys() {
     [addPastedElements, currentPage]
   );
 
+  /**
```

Comment:
@miina would you mind giving this PR a once over? The copy paste functionality is updated slightly for the right click menus, Samuel left helpful comments to give more context (as does the PR description).